geode-core/src/main/java/org/apache/geode/distributed/internal/ClusterDistributionManager.java

```diff
@@ -227,7 +248,9 @@ public class ClusterDistributionManager implements DistributionManager {
   private Map<InternalDistributedMember, Collection<String>> hostedLocatorsWithSharedConfiguration =
       Collections.emptyMap();
 
-  /** a map keyed on InternalDistributedMember, to direct channels to other systems */
+  /**
+   * a map keyed on InternalDistributedMember, to direct channels to other systems
+   */
   // protected final Map channelMap = CFactory.createCM();
```

Comment:
Cleanup of javadoc on a commented out member, please delete the member and javadoc.